.github/workflows/ci.yml

```diff
@@ -85,6 +85,21 @@ jobs:
           git diff --exit-code --color=always || ( echo "Terraform docs are
           out-of-date. See the diff above." && exit 1 )
 
+  tflint:
+    runs-on: ubuntu-latest
+
+    env:
+      TFLINT_CONFIG_FILE: "${{ github.workspace }}/.tflint.hcl"
```

Comment:
If TFLint runs in less than a second, consider adding it to the .githooks/pre-commit